### PR TITLE
Issue/241/create collected points view for admin

### PR DESCRIPTION
**Pull request by dbetka**

### CODE REVIEW

File: src/components/pages/admin/scoreboard.vue [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+      for (const pointId of user.collectedPointsIds) {
+        const point = this.$store.getters['event/getPointById'](pointId);
+        uCheck.isDefined(point) ? collectedPoints.push(point) : undefined;
+      }
+
+      return collectedPoints
+        .map(point => this.getCategoryById(point.pointCategory).pointValue)
+        .reduce((a, b) => (a + b), 0);
+    },
```
Comment (Bedpaw) [inline src/components/pages/admin/scoreboard.vue:110]: Maybe create method getPointValueById in store and then simple reduce in component

+ Duplication in all users store

File: src/store/all-users.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+export default {
+  namespaced: true,
+  state: {
+    users: [],
+  },
+  getters: {
+    users: state => state.users,
+    commonUsers (state, getters) {
+      return getters.users
```
Comment (Bedpaw) [inline src/store/all-users.js:12]: () => and remove return

File: src/store/all-users.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        .filter(user => user.accountType === ACCOUNT_TYPES.common);
+    },
+    userByUserField: (state) => userField => state.users.find(user => user.user === userField),
+    collectedPointsByUser: (state, getters, rootState, rootGetters) => user => {
+      const collectedPoints = [];
+
+      for (const pointId of user.collectedPointsIds) {
```
Comment (Bedpaw) [inline src/store/all-users.js:19]: Duplication in scoreboard

File: src/store/all-users.js (hunk 2 above)
Comment (dbetka) [reply src/store/all-users.js]: done

File: src/components/pages/admin/scoreboard.vue (hunk 1 above)
Comment (dbetka) [reply src/components/pages/admin/scoreboard.vue]: done

File: src/store/all-users.js (hunk 3 above)
Comment (dbetka) [reply src/store/all-users.js:17]: done

**Pull request merged**